scalableminds/webknossos
# Fix crash related to WEBGL_debug_renderer_info

[Context](https://scm.slack.com/archives/C5AKLAV0B/p1665156066659499)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- hard to test well; inspect code thoroughly instead. the outer try-catch should make the heuristic pretty robust :)

------
(Please delete unneeded items, merge only when none are left open)
